libs/remix-ui/run-tab/src/lib/run-tab.tsx

```diff
@@ -235,6 +235,16 @@ export function RunTabUI(props: RunTabProps) {
     }
   }, [runTab.popup])
 
+  useEffect(() => {
+    if (runTab.selectExEnv.includes('injected') &&
```

Comment:
what about wallets?

Reply:
I am not sure I understand what you mean by wallets.